include/ews/ews.hpp

```diff
@@ -8061,9 +8600,91 @@ namespace ews
             return xml().get_value_as_string("HasAttachments") == "true";
         }
 
-        // List of zero or more extended properties that are requested for
-        // this item
-        // TODO: get_extended_property
+        //! List of zero or more extended properties that are requested for
+        //! an item
+        std::vector<extended_property> get_extended_properties() const
+        {
+            std::vector<extended_property> vep;
+
+            for (auto top_node = xml().get_node("ExtendedProperty");
+                 top_node != nullptr; top_node = top_node->next_sibling())
+            {
+                if (std::string(top_node->name(), top_node->name_size()) !=
+                    "t:ExtendedProperty")
+                    continue;
+
+                for (auto ep_node = top_node->first_node(); ep_node != nullptr;
+                     ep_node = ep_node->next_sibling())
+                {
+                    auto ext_field_uri =
+                        extended_field_uri::from_xml_element(*ep_node);
+
+                    std::vector<std::string> values;
+                    ep_node = ep_node->next_sibling(); // go to value(es)
+
+                    if (std::string(ep_node->name(), ep_node->name_size()) ==
+                        "t:Value")
+                    {
+                        values.emplace_back(std::string(ep_node->value(),
+                                                        ep_node->value_size()));
+                    }
+                    else if (std::string(ep_node->name(),
+                                         ep_node->name_size()) == "t:Values")
+                    {
+                        for (auto node = ep_node->first_node(); node != nullptr;
+                             node = node->next_sibling())
+                        {
+                            values.emplace_back(
+                                std::string(node->value(), node->value_size()));
+                        }
+                    }
+                    vep.emplace_back(extended_property(ext_field_uri, values));
+                }
+            }
+            return vep;
+        }
+
```

Comment:
Yes, i realized what you mean and changed it. The term "docstring" is known in python (and PHP, i think) and was misleading me.
Everything is done.